### ISSUE TITLE
Auto Update not working

### ISSUE THREAD

**Issue by JCVanderbilt** [meganz/MEGAsync#688]
Currently on MEGAsync 4.6.1 64-bit, open the interface, click "Check for Updates", it finds an update but the update fails.  It looks like it's failing because MEGAsync.exe is in use maybe?  Also tried it while antivirus was disabled, same result.

From MEGAsync.log:

03/13-15:43:00.049987 41456 INFO Starting update check [:-1]
03/13-15:43:00.056902 41456 DBG  Downloading updated file from http://g.static.mega.co.nz/upd/wsync64/v.txt?PHQGHUMEAY [:-1]
03/13-15:43:00.657357 41456 DBG  Reading update info [:-1]
03/13-15:43:00.657382 41456 INFO Update available! Current version: 4601  Update version: 4605 [:-1]
03/13-15:43:00.657947 41456 INFO File already installed: api-ms-win-core-console-l1-1-0.dll [:-1]
03/13-15:43:00.658191 41456 INFO File already installed: api-ms-win-core-datetime-l1-1-0.dll [:-1]
03/13-15:43:00.658403 41456 INFO File already installed: api-ms-win-core-debug-l1-1-0.dll [:-1]
03/13-15:43:00.658611 41456 INFO File already installed: api-ms-win-core-errorhandling-l1-1-0.dll [:-1]
03/13-15:43:00.658821 41456 INFO File already installed: api-ms-win-core-file-l1-1-0.dll [:-1]
03/13-15:43:00.659028 41456 INFO File already installed: api-ms-win-core-file-l1-2-0.dll [:-1]
03/13-15:43:00.659226 41456 INFO File already installed: api-ms-win-core-file-l2-1-0.dll [:-1]
03/13-15:43:00.659424 41456 INFO File already installed: api-ms-win-core-handle-l1-1-0.dll [:-1]
03/13-15:43:00.659622 41456 INFO File already installed: api-ms-win-core-heap-l1-1-0.dll [:-1]
03/13-15:43:00.659819 41456 INFO File already installed: api-ms-win-core-interlocked-l1-1-0.dll [:-1]
03/13-15:43:00.660025 41456 INFO File already installed: api-ms-win-core-libraryloader-l1-1-0.dll [:-1]
03/13-15:43:00.660232 41456 INFO File already installed: api-ms-win-core-localization-l1-2-0.dll [:-1]
03/13-15:43:00.660432 41456 INFO File already installed: api-ms-win-core-memory-l1-1-0.dll [:-1]
03/13-15:43:00.660630 41456 INFO File already installed: api-ms-win-core-namedpipe-l1-1-0.dll [:-1]
03/13-15:43:00.660852 41456 INFO File already installed: api-ms-win-core-processenvironment-l1-1-0.dll [:-1]
03/13-15:43:00.661061 41456 INFO File already installed: api-ms-win-core-processthreads-l1-1-0.dll [:-1]
03/13-15:43:00.661262 41456 INFO File already installed: api-ms-win-core-processthreads-l1-1-1.dll [:-1]
03/13-15:43:00.661461 41456 INFO File already installed: api-ms-win-core-profile-l1-1-0.dll [:-1]
03/13-15:43:00.661662 41456 INFO File already installed: api-ms-win-core-rtlsupport-l1-1-0.dll [:-1]
03/13-15:43:00.661861 41456 INFO File already installed: api-ms-win-core-string-l1-1-0.dll [:-1]
03/13-15:43:00.662063 41456 INFO File already installed: api-ms-win-core-synch-l1-1-0.dll [:-1]
03/13-15:43:00.662261 41456 INFO File already installed: api-ms-win-core-synch-l1-2-0.dll [:-1]
03/13-15:43:00.662648 41456 INFO File already installed: api-ms-win-core-sysinfo-l1-1-0.dll [:-1]
03/13-15:43:00.662865 41456 INFO File already installed: api-ms-win-core-timezone-l1-1-0.dll [:-1]
03/13-15:43:00.663065 41456 INFO File already installed: api-ms-win-core-util-l1-1-0.dll [:-1]
03/13-15:43:00.663360 41456 INFO File already installed: api-ms-win-crt-conio-l1-1-0.dll [:-1]
03/13-15:43:00.663569 41456 INFO File already installed: api-ms-win-crt-convert-l1-1-0.dll [:-1]
03/13-15:43:00.663766 41456 INFO File already installed: api-ms-win-crt-environment-l1-1-0.dll [:-1]
03/13-15:43:00.663969 41456 INFO File already installed: api-ms-win-crt-filesystem-l1-1-0.dll [:-1]
03/13-15:43:00.664172 41456 INFO File already installed: api-ms-win-crt-heap-l1-1-0.dll [:-1]
03/13-15:43:00.664381 41456 INFO File already installed: api-ms-win-crt-locale-l1-1-0.dll [:-1]
03/13-15:43:00.664595 41456 INFO File already installed: api-ms-win-crt-math-l1-1-0.dll [:-1]
03/13-15:43:00.664825 41456 INFO File already installed: api-ms-win-crt-multibyte-l1-1-0.dll [:-1]
03/13-15:43:00.665132 41456 INFO File already installed: api-ms-win-crt-private-l1-1-0.dll [:-1]
03/13-15:43:00.665325 41456 INFO File already installed: api-ms-win-crt-process-l1-1-0.dll [:-1]
03/13-15:43:00.665529 41456 INFO File already installed: api-ms-win-crt-runtime-l1-1-0.dll [:-1]
03/13-15:43:00.665732 41456 INFO File already installed: api-ms-win-crt-stdio-l1-1-0.dll [:-1]
03/13-15:43:00.665948 41456 INFO File already installed: api-ms-win-crt-string-l1-1-0.dll [:-1]
03/13-15:43:00.666146 41456 INFO File already installed: api-ms-win-crt-time-l1-1-0.dll [:-1]
03/13-15:43:00.666343 41456 INFO File already installed: api-ms-win-crt-utility-l1-1-0.dll [:-1]
03/13-15:43:00.706823 41456 INFO File already installed: avcodec-58.dll [:-1]
03/13-15:43:00.714044 41456 INFO File already installed: avformat-58.dll [:-1]
03/13-15:43:00.716160 41456 INFO File already installed: avutil-56.dll [:-1]
03/13-15:43:00.716540 41456 INFO File already installed: cares.dll [:-1]
03/13-15:43:00.717612 41456 INFO File already installed: concrt140.dll [:-1]
03/13-15:43:00.717899 41456 INFO File already installed: iconengines/qsvgicon.dll [:-1]
03/13-15:43:00.718159 41456 INFO File already installed: imageformats/qgif.dll [:-1]
03/13-15:43:00.718438 41456 INFO File already installed: imageformats/qicns.dll [:-1]
03/13-15:43:00.718690 41456 INFO File already installed: imageformats/qico.dll [:-1]
03/13-15:43:00.719813 41456 INFO File already installed: imageformats/qjpeg.dll [:-1]
03/13-15:43:00.720066 41456 INFO File already installed: imageformats/qsvg.dll [:-1]
03/13-15:43:00.720298 41456 INFO File already installed: imageformats/qtga.dll [:-1]
03/13-15:43:00.721586 41456 INFO File already installed: imageformats/qtiff.dll [:-1]
03/13-15:43:00.721834 41456 INFO File already installed: imageformats/qwbmp.dll [:-1]
03/13-15:43:00.723282 41456 INFO File already installed: imageformats/qwebp.dll [:-1]
03/13-15:43:00.731475 41456 INFO File already installed: libcrypto-1_1-x64.dll [:-1]
03/13-15:43:00.733248 41456 INFO File already installed: libcurl.dll [:-1]
03/13-15:43:00.735165 41456 INFO File already installed: libssl-1_1-x64.dll [:-1]
03/13-15:43:00.735386 41456 INFO File already installed: MEGA Website.url [:-1]
03/13-15:43:00.889730 41456 INFO File already installed: msvcp140_1.dll [:-1]
03/13-15:43:00.890309 41456 INFO File already installed: msvcp140_2.dll [:-1]
03/13-15:43:00.890591 41456 INFO File already installed: msvcp140_atomic_wait.dll [:-1]
03/13-15:43:00.890794 41456 INFO File already installed: msvcp140_codecvt_ids.dll [:-1]
03/13-15:43:00.892148 41456 INFO File already installed: msvcp140.dll [:-1]
03/13-15:43:00.896153 41456 INFO File already installed: platforms/qwindows.dll [:-1]
03/13-15:43:00.896488 41456 INFO File already installed: Qt5Concurrent.dll [:-1]
03/13-15:43:00.910604 41456 INFO File already installed: Qt5Core.dll [:-1]
03/13-15:43:00.926060 41456 INFO File already installed: Qt5Gui.dll [:-1]
03/13-15:43:00.929700 41456 INFO File already installed: Qt5Network.dll [:-1]
03/13-15:43:00.930678 41456 INFO File already installed: Qt5Svg.dll [:-1]
03/13-15:43:00.943908 41456 INFO File already installed: Qt5Widgets.dll [:-1]
03/13-15:43:00.944686 41456 INFO File already installed: Qt5Xml.dll [:-1]
03/13-15:43:00.944868 41456 INFO File already installed: qt.conf [:-1]
03/13-15:43:00.949270 41456 INFO File already installed: styles/qwindowsvistastyle.dll [:-1]
03/13-15:43:00.950839 41456 INFO File already installed: swresample-3.dll [:-1]
03/13-15:43:00.953663 41456 INFO File already installed: swscale-5.dll [:-1]
03/13-15:43:00.956071 41456 INFO File already installed: ucrtbase.dll [:-1]
03/13-15:43:00.958142 41456 INFO File already installed: vccorlib140.dll [:-1]
03/13-15:43:00.958693 41456 INFO File already installed: vcomp140.dll [:-1]
03/13-15:43:00.958980 41456 INFO File already installed: vcruntime140_1.dll [:-1]
03/13-15:43:00.959378 41456 INFO File already installed: vcruntime140.dll [:-1]
03/13-15:43:01.109383 41456 INFO File already downloaded: MEGAsync.exe [:-1]
03/13-15:43:01.115612 41456 INFO File already downloaded: MEGAupdater.exe [:-1]
03/13-15:43:01.117899 41456 INFO File already downloaded: ShellExtX64.dll [:-1]
03/13-15:43:01.118791 41456 INFO File already downloaded: uninst.exe [:-1]
03/13-15:43:01.119386 41456 INFO Applying update... [:-1]
**03/13-15:43:01.155159 41456 ERR  Error installing file: MEGAsync.exe in C:/Users/JCVanderbilt/AppData/Local/MEGAsync/MEGAsync.exe [:-1]**
03/13-15:43:01.155163 41456 INFO Uninstalling update... [:-1]
03/13-15:43:01.155501 41456 INFO File restored: MEGAsync.exe [:-1]
03/13-15:43:01.155502 41456 INFO Update uninstalled [:-1]
03/13-15:43:01.155505 41456 INFO Update task finished. No updates available [:-1]

**Comment by angelds-mega** [comment]
Hello. We are not able to reproduce the issue. 

In order to keep investigating, can you please give us the following informations: 

1 -Which OS version are you using please? 
2 - System configuration details, file system type, any extra detail.
3.- Was the 4.6.1 64-bit the first version installed or were you able to  update in the past?

Thank you.

**Comment by JCVanderbilt** [comment]
Now that you asked for the second item I've figured out what it is.

My "%LocalAppData%\Mega Limited" directory is a directory symbolic link to a SSD raid I use to store things I want to have a backup of if I ever lose my C: drive.  It points to D:\Libraries\AppData\Local\Mega Limited.  Once I deleted the symlink and moved the folder back to "%LocalAppData%\Mega Limited" the update happened when I relaunched the app.

Maybe something about the update process doesn't like symbolic links or the fact that the source is on C: and the destination is on a different drive.
